### ISSUE TITLE
AdvancedPOP3Client.retrieve docstring is insufficient

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6250]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6250|
|Type|defect|
|Created|2013-01-10 19:26:24Z|

`AdvancedPOP3Client.retrieve()` doesn't document:

1. that it uses 0-indexing.
2. The semantics of the lines argument.

<details><summary>Searchable metadata</summary>

```
trac-id__6250 6250
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__mail mail
keywords__documentation documentation
time__1357845984000000 1357845984000000
changetime__1379329377000000 1379329377000000
version__None None
owner__shira shira

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="staceysern's avatar" src="https://avatars.githubusercontent.com/u/3631162?s=50" width="50" height="50">](https://github.com/staceysern)|@staceysern set owner to @staceysern|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="staceysern's avatar" src="https://avatars.githubusercontent.com/u/3631162?s=50" width="50" height="50">](https://github.com/staceysern)<a name="note_2"></a>|@staceysern set status to `closed`|
|-|-|

This is addressed in ticket [#6653](https://github.com/twisted/twisted/issues/6653).
